debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/SourceInfo.java

```diff
@@ -229,7 +200,7 @@ public BsonTimestamp lastOffsetTimestamp(String replicaSetName) {
      * @see #schema()
      */
     public Struct lastOffsetStruct(String replicaSetName, CollectionId collectionId) {
-        return offsetStructFor(replicaSetName, collectionId.namespace(), positionsByReplicaSetName.get(replicaSetName),
+        return offsetStructFor(replicaSetName, collectionId, positionsByReplicaSetName.get(replicaSetName),
```

Comment:
Why is this changing here?

Reply:
@gunnarmorling I put it inline with other connectors. Namespace is db + collection. So now we are sendnig db field and collection field.

Reply:
I see; is this considered in the code that reads back the offset? I.e. does it support old and new format?

Reply:
Btw. that method ought to be renamed, it doesn't return a (KC) offset but rather a source info struct.

Reply:
@gunnarmorling Renamed, also `ns` is not part of offset but sourcinfo only so no need to handle reload